### DISCUSSION TITLE
Dashboard/lovelace Music Assistant Support

### DISCUSSION THREAD

**Discussion by kksligh** [music-assistant/support#954] [Feature requests and ideas]
### Current Status
As at August 2025 various community developed cards are available and are listed here https://www.music-assistant.io/community-extensions/

### Original Post 
I love this platform and see a ton of potential here. Id really like to see a front end card or support created for this application. Especially something mobile friendly.  I've tried using an iframe and other workarounds but to no avail. It could literally be the same interface design, just create a compatible front end card that can be used within the dashboard. I understand it can be used as a tab with HA but that breaks users outside the front end experience and puts them into the HA application which is not good UX, especially if you've invested a lot of time building out different dashboards for different devices and users. 

Thank you for all this wonderful work. Be glad to test and support in any way.

**Comment by OzGav** [comment]
I’ve asked the front end dev what he thinks of your request.

**Reply by gohigher14** [reply]
> I love this platform and see a ton of potential here. Id really like to see a front end card or support created for this application. Especially something mobile friendly. I've tried using an iframe and other workarounds but to no avail. It could literally be the same interface design, just create a compatible front end card that can be used within the dashboard. I understand it can be used as a tab with HA but that breaks users outside the front end experience and puts them into the HA application which is not good UX, especially if you've invested a lot of time building out different dashboards for different devices and users.
> 
> Thank you for all this wonderful work. Be glad to test and support in any way.

Exactly , it would be so so nice to have this on a lovelace card.  Hope that the front end dev want to do this.

**Comment by GizmoGoody** [comment]
@OzGav Is this request a dupe of this https://github.com/music-assistant/support/discussions/415 which has twice the votes already?

Either way, I gave both a +1.

I've been struggling with how to make Music Assistant work in my multi-speaker setup and existing UI. I'm not crazy with having to use the sidebar since I hide it for the rest of my HA UX.

After trying to get one of the three primary media player cards...

![image](https://user-images.githubusercontent.com/46235745/210805027-681320f9-96d4-41fe-9f33-938bba35cab8.png)

...to do everything I need, I kept finding all of them are missing at least one feature:

- Media Control: shuffle and repeat, speaker grouping, and all the customization available in the other two
- Mini Media Player: media library browse* and the look and feel many of us love with Mushroom Cards
- Mushroom Media Card: source or media library browse* and speaker grouping

*See [here](https://github.com/kalkih/mini-media-player/issues/435#issuecomment-1092157117) and previous comments for more information on why this is still not possible without some ridiculous card_mod'ing.

It would be great if @marcelveldt, @kalkih, and @piitaya could team up to create one Media Player card to rule them all!

**Reply by OzGav** [reply]
Some great ideas here. The next version of MA will have an improved interface. I am not sure whether it will meet all your requirements and I don’t think it will be a separate card at this time. Maybe like in that other thread you could mock up some ideas that might serve as inspiration to the front end devs…?

**Reply by GizmoGoody** [reply]
You bet. The dashboard view I'm piecing together could probably double as a mock-up showing the limits of how far a non-coder can take the current options without further frontend development and integration.

**Comment by Swerfer** [comment]
> I love this platform and see a ton of potential here. Id really like to see a front end card or support created for this application. Especially something mobile friendly. I've tried using an iframe and other workarounds but to no avail. It could literally be the same interface design, just create a compatible front end card that can be used within the dashboard. I understand it can be used as a tab with HA but that breaks users outside the front end experience and puts them into the HA application which is not good UX, especially if you've invested a lot of time building out different dashboards for different devices and users.
> 
> Thank you for all this wonderful work. Be glad to test and support in any way.

You can use MA on a Dashboard in a IFrame card. I used Edge and right clicked on the MA dashboard and clicked 'Inspect'. There I searched for the IFrame url, see screenshot:

![image](https://github.com/music-assistant/support/assets/14137117/bec86634-1e4c-431f-ae9f-ab138193a481)

Result (top-right):

![image](https://github.com/music-assistant/support/assets/14137117/eb9da405-4cc8-4c36-8910-2475d2e1680c)

**Reply by mbrevda** [reply]
This only works for users who are Administrators. If using a restricted user, this solution wont work

**Comment by marcelveldt** [comment]
Development of a custom card for HA is definitely on the wishlist but I do not have time to work on that any time soon. It will definitely not have any priority before 2.0 stable is released. After that we can have a look if there's enough user demand. So until then you will have to do with using MA's own interface or just use default HA cards. 

What I use myself is a media card for the group player and then entity rows for the child players within the group for easy power and volume control. i always play to the group player(s) and just power on/off players in the rooms.

**Comment by JohnnyCarcinogen** [comment]
Would you mind elaborating on how you use a "media card for the group player"? I've got a Sonos system, and that would be great if it was all in one, but so far all I see are the individual speakers.

**Reply by codemonkey2k5** [reply]
You can create and use groups by adding a "Universal Group Player" from the add player section in settings.

**Comment by jazzyisj** [comment]
My media UI might be of interest here.  I use variations of the mini-card depending on the player state.  

If a single player is playing a stream, the player opens up and displays the controls.  If a group is playing, the sub members only show the volume control and the group media player opens up and shows the controls.

It isn't a simple setup.  You'll have to set up some template sensors to help determine the player states and use a whole whack of conditional cards.  But it does work well once it's set up.

In the screenshot below I have both a group and a single media player playing different streams so you can see how it works.

If anybody is curious I can provide more details.

EDIT: Details for that Play Media card on the bottom left.
https://github.com/orgs/music-assistant/discussions/2990

<img width="1280" alt="image" src="https://github.com/user-attachments/assets/806f8396-504b-4c26-b8c5-d4463744409a">

**Reply by aschwalb** [reply]
This is the exact issue I have. I have built a nice UI for my wife but in order to use MA she has to know to drop out of the ui and find the MA on the left.  Not good UX.  I have a whole home audio (HTD) with audio zone controls but not way to easily select music.  I like what you have started here and would like to replicate.  I would live the MASS UI just embedded in my normal dashboard or sections of it.

**Comment by potelux** [comment]
I detailed an idea ("super cards") that might make a custom dashboard card easier [here](https://github.com/orgs/music-assistant/discussions/2601#discussioncomment-11326049).

Additionally, I created [a feature request in Home Assistant](https://community.home-assistant.io/t/dashboard-card-blueprints/709559) to allow this type of card.

If you think the "super card" idea is a valid solution to this problem, let me know.  I am considering making it my holiday project and would like to know if there is genuine interest.

**Comment by felipetoffolo1** [comment]
I do think that just embedding MASS is probably the easiest solution.

<img width="1671" alt="Screenshot 2024-11-22 at 11 38 12" src="https://github.com/user-attachments/assets/51d8f7cd-f4c3-43ae-92b1-a6c5cfbfb8f4">
<img width="1671" alt="Screenshot 2024-11-22 at 11 38 02" src="https://github.com/user-attachments/assets/e7d998e3-6a30-4a14-86fe-698a79586c1b">

You can even use single cards, or multiple cards for other things.
Trying to add extra customization to the frontend might make it possible to to a lot.

- Supporting routes to be served without the "Main Frame". Ex: having a `/browser?standalone=true&player_target=my_playes` where it shows the  browser view without all the navigation around. That would make it possible to add just that to a card, and allow people do compose a dashboard on different ways
- Supporting themes. Basically just allow for community themes for people to go crazy.

I'm a frontend eng. So I can help with that. I'm not sure how the contribution process work for the project. 
But I think that I could provide the route/query param solution with relative easy, if that's something the team here would like.

**Reply by OzGav** [reply]
If you join the discord server there is a devs support channel where you can ask questions

**Reply by marcelveldt** [reply]
All help is welcome, especially frontend!
Here is the frontend repo, dev instructions included. If you are somewhat familiar with typescript and Vue, you should be going within minutes.

https://github.com/music-assistant/frontend

**Reply by jbega** [reply]
> * Supporting routes to be served without the "Main Frame". Ex: having a `/browser?standalone=true&player_target=my_playes` where it shows the  browser view without all the navigation around. That would make it possible to add just that to a card, and allow people do compose a dashboard on different ways

@felipetoffolo1 Would you be able to provide more details on this? Thanks

**Reply by OzGav** [reply]
There is not much more to say than what is described. It will be added in an upcoming release

**Reply by missanhome** [reply]
Hi Guys,

I've just installed MA a few hours ago, linked Tidal and I must say I fell in love onto this project. It's just what I was looking for!
Since then I'm searching the internet on how to achieve exactly what @felipetoffolo1 described (I'm not a developer, unfortunately). It would be just perfect on my tablet screen (my main HA control panel). A separate view with the main content under wallpanel to have a screensaver while casting music.

So I just came to say thank you all and +1 to @felipetoffolo1 design / idea, I believe there are many others (in silence) who will love this simple but great full view.

**Reply by OzGav** [reply]
This is implemented in the beta version of the server

**Reply by jazzyisj** [reply]
@missanhome - It was added a couple of releases ago but not really documented anywhere yet that I have been able to find. 

https://github.com/music-assistant/frontend/pull/745

This is the only way I've been able  to get to work.  Note, it's on the webserver, I don't think this will work with the ingress url.

`http://homeassistant.local:8095/#/playlists?frameless=true`

But I can't figure out how to get it to target a player or be able to select a player.  There is supposed to be a "player" parameter but I wasn't able to get it to work.  Tried all kinds of varations (player name as camel, quoted, slugified, entity_id, mass player id). All the tabs work though (artists, albums, tracks etc.)

`http://homeassistant.local:8095/#/artists?frameless=true`

```
title: "Music Assistant"
path: mass
type: sections
subview: true
max_columns: 3
dense_section_placement: true
sections:
  - type: grid
    cards:
      - !include /config/ui/card/media/media_player/media_players_group.yaml

  - type: grid
    cards:
      - !include /config/ui/card/media/media_player/media_players_speaker.yaml

  - type: grid
    cards:
      - type: iframe
        grid_options:
          rows: 12
        url: http://homeassistant.local:8095/#/playlists?frameless=true
```
The double scroll bars isn't ideal...

<img width="1278" alt="image" src="https://github.com/user-attachments/assets/91099b2e-2334-45de-97ab-841fcf49cec3" />

<img width="380" alt="image" src="https://github.com/user-attachments/assets/d3515c95-2cd2-484f-be36-05af284c375f" />

**Reply by OzGav** [reply]
Info is now in the new beta version of the docs. Double scroll bar has been noted

**Reply by felipetoffolo1** [reply]
@jazzyisj @OzGav there is some kind of cache in place I think.

I was able to use 
```
type: horizontal-stack
cards:
  - type: iframe
    url: >-
      http://192.168.86.38:8095/#/home?player=hifiberry&showFullscreenPlayer=true
    aspect_ratio: 100%
  - type: iframe
    url: http://192.168.86.38:8095/#/browse?player=hifiberry&frameless=true
    aspect_ratio: 100%
    visibility:
      - condition: screen
        media_query: "(min-width: 768px)"
```

but if I change the player manually and reload I need to open the devtools and select "disable cache" in the network tab.
The same does not happen when I run in dev, so maybe we have some cache layer somewhere?

**Reply by OzGav** [reply]
@felipetoffolo1 wouldn’t this be a HA or browser problem though? Unless you can force disable the cache?

**Reply by felipetoffolo1** [reply]
@OzGav I was able to reproduce outside of HA.

I think it's a vue-router thing

**Reply by felipetoffolo1** [reply]
@OzGav here https://github.com/music-assistant/frontend/pull/808

**Reply by OzGav** [reply]
> The double scroll bars isn't ideal...

Fixed in 2.4.0b14 Thanks @felipetoffolo1 !!!

**Reply by brochond** [reply]
@felipetoffolo1 Thanks for these suggestions!

I have a question, though—would it be possible to directly display the queue items on screen using the following URL?
http://IP_ADDRESS:8095/#/home?player=my_player&showFullscreenPlayer=true

I tried adding the path parameter showQueueItems=true, but fortunately, it doesn’t work. Indeed, using a path variable directly in the Vue store is not effective.

For reference, here’s a screenshot of the expected view: 

<img width="1356" alt="image" src="https://github.com/user-attachments/assets/ead5c493-e435-4885-81bd-78d77d98755f" />

**Comment by jbega** [comment]
@felipetoffolo1 , @OzGav, @jazzyisj  Could that cache you mentioned have existed before the update? I've noticed a few weeks

- If using -> http://xx.xxx.xx.xx/#/artists?frameless=true&targetPlayer=Cocina  (Whether it's an artist, a playlist, or any selection other than 'home')   -> show last (or none)player page

-- If using first -> http://xx.xxx.xx.x/#/home?frameless=true&targetPlayer=Cocina  and then use  http://xx.xxx.xx.x/#/artists?frameless=true&targetPlayer=Cocina -> shows correctly player cocina web

**Reply by felipetoffolo1** [reply]
@jbega we are not using `targetPlayer` anymore. Just `player`

**Reply by jbega** [reply]
@felipetoffolo1 Yes, I'm aware that it's no longer used, that's why I mentioned that "before" the update it worked like that and had that behavior. I was saying it in case you were seeing the same behavior with the new way, just in case it was helpful.

@missanhome I see that when you have the card/panel horizontally, the player's action square/button (rectangle below) disappears... it happens to me too, does anyone know how to make it not disappear? It's a convenient way to switch players without many clicks


**Besides, I don't know if this has happened to anyone else:

1- In dashboard panel mode config (Ha web-PC), the frameless function doesn't work. if its set on horizontal layout, it works with vertical menu

@felipetoffolo1 example code
![image](https://github.com/user-attachments/assets/589c6740-091e-42f8-9907-634d67b347db)




2- I've never been able to hide the horizontal menu, only the vertical one (although it takes a couple of clicks). 

@jazzyisj example code
![image](https://github.com/user-attachments/assets/4fb9c42f-e7a8-424d-81e2-285ccd117081)

**Reply by missanhome** [reply]
@jbega,
If you don't tell, I won't even recognize the disappearing Player box. 😄 
However when I checked yesterday I noticed that it seems the let's call "What's playing" view is higher. If I set the aspect ratio to e.g. 75% the Player box appeared. This unfortunately affects the main page view as well, but as a workaround it may be good for you.

It would be a very good feature to have some kind of scaling/resizing of the internal content. I mean here do downsize the content to see the artists/albums in smaller scale, having more on the main page..

If this is in place, it would be very much appreciated if someone would tell how to achieve that. Thanks.

**Reply by jbega** [reply]
> @jbega, If you don't tell, I won't even recognize the disappearing Player box. 😄 However when I checked yesterday I noticed that it seems the let's call "What's playing" view is higher. If I set the aspect ratio to e.g. 75% the Player box appeared. This unfortunately affects the main page view as well, but as a workaround it may be good for you.

Thanks, I'll try it that way

> It would be a very good feature to have some kind of scaling/resizing of the internal content. I mean here do downsize the content to see the artists/albums in smaller scale, having more on the main page..

This would be great because in a Ma dashboard we're testing, when viewed on a tablet, the elements inside the iframe (e.g., artist photo) are quite large (unlike the menus, which are the correct size

https://github.com/orgs/music-assistant/discussions/2601#discussioncomment-11776538

https://github.com/orgs/music-assistant/discussions/2601#discussioncomment-11784742

The videos, which are captured directly from the PC screen, maintain the correct aspect ratio and scale when viewed

**Comment by missanhome** [comment]
First of all, thank you for your comments. Here's a little update on how it went after a few hours of trial and error:

- No matter how I tried I could not put the MA's Home screen onto a HA card or a HA view if the server was installed as a Home Assistant Add-on (just as @jazzyisj mentioned).
- Finally I ended up installing the server as a Docker image to my Synology NAS, but at my workplace I could not get it work remotely (first try with Docker, no clue how to expose to outside world).
- At home on local network I could configure the music provider (Tidal) and player (Chromecast) easily on http://nas_local_address:8095.
- Then I started to try adding it as an iFrame card using different View configurations, layouts, aspect ratio, various earlier mentioned ?frameless=true URLs.

I've ended up with this:

![MA1](https://github.com/user-attachments/assets/469494e0-6e95-4af7-b462-bc263dd8765f)

![MA2](https://github.com/user-attachments/assets/28c99f87-8ea8-4c80-9cb3-facf223bef68)

Devices I use: RPi 4 with Home Assistant OS, wall mounted Android tablet with HA Companion App as the main control panel, Google Nest Audio to cast music to.

It's definitely not the most sophisticated and nicest solution, but it is what I wanted to achieve: having the full functionality of MA on a HA dashboard view, under Wallpanel's screensaver umbrella, preferably on full screen on my tablet.
It is working locally only, though I did not think on casting music to home during holiday, so it's OK from my side.

If anyone is interested, this is the config I used:
- prerequisite: layout-card from HACS
- HA View configuration: Horizontal (layout-card)
- Card configuration, where 192.168.1.6 is my NAS's local IP, aspect ratio and width correspond with the tablet's size:

```
type: custom:layout-card
layout_type: custom:vertical-layout
cards:
  - type: iframe
    url: http://192.168.1.6:8095
    aspect_ratio: 55%
layout:
  width: 1320
```

Sorry for the long post, certainly for most of you this is obvious, but for noobs as myself it may be useful at least as a starting point.

**Reply by jazzyisj** [reply]
I didn't say it didn't work with the addon, I said it doesn't work if you access it with ingress url.  You have to turn on the music assistant web server and use that url.

<img width="513" alt="image" src="https://github.com/user-attachments/assets/6e33c229-7402-4327-8fff-ed888f464417" />

**Reply by missanhome** [reply]
Ah... all this hassle with the Docker while it would work with a simple setting... 🥲
Thanks for adding this info. I will not turn back at this point, but for others it may be useful. At least I learned something new (Docker).

**Comment by Flipsoo** [comment]
hey there, 
i also struggle to get an easier interface for music assistant that nicely fits my styling needs. 

I came up with a really messy workaround, but i wanted to share it with the community if this helps someone somehow. 

First of all i setup a sensor in NodeRed, this is a snippet for the playlist and can be adapted for all lists. I basically just extract the needed values for the lovelance cards here. 

``
   [{"id":"61de42d8838244f9","type":"ha-sensor","z":"6a93e608efc947bf","g":"a0362e50fd5cc18f","name":"Music Assistant Most Played","entityConfig":"43d4ca3adf4b5255","version":0,"state":"1","stateType":"str","attributes":[{"property":"object","value":"payload2","valueType":"msg"},{"property":"tracks","value":"payload3","valueType":"msg"},{"property":"favs","value":"payload4","valueType":"msg"},{"property":"playlists","value":"payload5","valueType":"msg"}],"inputOverride":"allow","outputProperties":[],"x":760,"y":2440,"wires":[[]]},{"id":"2b09ac48ff90e644","type":"api-call-service","z":"6a93e608efc947bf","g":"a0362e50fd5cc18f","name":"","server":"1265227d.1efcae","version":7,"debugenabled":false,"action":"music_assistant.get_library","floorId":[],"areaId":[],"deviceId":[],"entityId":[],"labelId":[],"data":" {\"config_entry_id\": \"CONFIG_ENTRY_NAME\",\t     \"limit\": 35,\t    \"offset\": 0,\t    \"media_type\": \"playlist\",\t    \"order_by\": \"random\"\t  }\t","dataType":"jsonata","mergeContext":"","mustacheAltTags":false,"outputProperties":[{"property":"payload","propertyType":"msg","value":"","valueType":"results"}],"queue":"none","blockInputOverrides":true,"domain":"music_assistant","service":"get_library","x":760,"y":2360,"wires":[["6039fa92974af62f"]]},{"id":"6039fa92974af62f","type":"function","z":"6a93e608efc947bf","g":"a0362e50fd5cc18f","name":"function 13","func":"// Initialize an array to store the simplified track objects\nlet simplifiedTracks = [];\n\n// Loop through each track in msg.payload.items\nmsg.payload.items.forEach((trackData) => {\n    // Extract the required elements for each track\n    let trackName = trackData.name || \"Unknown Track Name\";\n    let artistName = trackData.artists && trackData.artists.length > 0\n        ? trackData.artists[0].name\n        : \"Unknown Artist\";\n    let trackUri = trackData.uri || \"Unknown URI\";\n    let trackImage = trackData.image || \"Unknown Track Image\";\n\n    // Add the simplified track data to the array\n    simplifiedTracks.push({\n        track_name: trackName,\n        artist_name: artistName,\n        track_uri: trackUri,\n        track_image: trackImage\n    });\n});\n\n// Set the output payload to the array of simplified track objects\nmsg.payload5 = simplifiedTracks;\n\n// Return the modified message object\nreturn msg;","outputs":1,"timeout":0,"noerr":0,"initialize":"","finalize":"","libs":[],"x":870,"y":2360,"wires":[["a4c6965a7ac83928","61de42d8838244f9"]]},{"id":"43d4ca3adf4b5255","type":"ha-entity-config","server":"1265227d.1efcae","deviceConfig":"","name":"Music Assistant Most Played","version":"6","entityType":"sensor","haConfig":[{"property":"name","value":"Music Assistant Most Played"},{"property":"icon","value":""},{"property":"entity_picture","value":""},{"property":"entity_category","value":""},{"property":"device_class","value":""},{"property":"unit_of_measurement","value":""},{"property":"state_class","value":""}],"resend":false,"debugEnabled":false},{"id":"1265227d.1efcae","type":"server","name":"Home Assistant","version":5,"addon":true,"rejectUnauthorizedCerts":true,"ha_boolean":"y|yes|true|on|home|open","connectionDelay":false,"cacheJson":false,"heartbeat":false,"heartbeatInterval":"30","areaSelector":"friendlyName","deviceSelector":"id","entitySelector":"id","statusSeparator":"at: ","statusYear":"hidden","statusMonth":"short","statusDay":"numeric","statusHourCycle":"h23","statusTimeFormat":"h:m","enableGlobalContextStore":true}]``

With this sensor i then use an auto-entity card to populate single custom-button-cards that are clickable to play the media. 

`

     - type: custom:auto-entities
        show_empty: true
        card:
          type: grid
          square: false
          columns: 3
        card_param: cards
        filter:
          template: >
            {%- set player =  states.input_select.media_player_selector.state
            -%}  {% for element in
            states.sensor.music_assistant_most_played.attributes.playlists[:25]
            -%}
            {{
              {
                "aspect_ratio": "1/1.5",
                "color": "var(--primary-color)",
                "label": '<div id="highlight">'+element.track_name,
                "name": '<div id="highlight">'+element.artist_name,
                "show_icon": false,
                "show_label": true,
                "show_name": true,
                "state": [
                  {
                    "styles": {
                      "card": [
                        {
                          "filter": "grayscale(100%)"
                        }
                      ]
                    },
                    "value": "unavailable"
                  }
                ],
                "styles": {
                  "name": [
                    {
                      "color": "black",
                      "font-weight": "1000",
                      "font-size": "12px",
                      "line-break": "auto",
                      "white-space": "pre-line"
                    }
                  ],
                  "label": [
                    {
                      "color": "black",
                      "font-weight": "1000",
                      "font-size": "14px",
                      "line-break": "auto",
                      "white-space": "pre-line"
                    }
                  ]
                },
                "tap_action": {
                  "action": "call-service",
                  "confirmation": {
                    "text": "[[[ return 'Play track?' ]]]"
                  },
                  "service": "media_player.play_media",
                  "service_data": {
                    "media_content_id": element.track_uri,
                    "media_content_type": "playlist",
                    "enqueue": "replace",
                    "entity_id": player,
                  }
                },
                "hold_action": {
                  "action": "call-service",
                  "service": "media_player.play_media",
                  "service_data": {
                    "media_content_id": element.track_uri,
                    "media_content_type": "playlist",
                    "enqueue": "play",
                    "radio_mode": "true",
                    "entity_id": player
                  }
                },
                "double_tap_action": {
                  "action": "call-service",
                  "service": "media_player.play_media",
                  "service_data": {
                    "media_content_id": element.track_uri,
                    "media_content_type": "playlist",
                    "enqueue": "add",
                    "entity_id": player
                  }
                },
                "type": "custom:button-card",
                "extra_styles": "@keyframes fade-in { 0% {opacity: 0;} 100% {opacity: 1; visibility: visible;} } @media (hover: hover) { ha-card:hover { opacity: 0.5 !important; } }",
                "card_mod": {
                  "style": "ha-card { background: url("+element.track_image+"); background-size: 150%; background-repeat: no-repeat; background-position: center; background-size: cover; background-color: var(--ha-card-background) !important; } #highlight { display: inline-block; z-index: -1; background-color: #FFFF; }"
                }
              }
            }},

            {%- endfor %}
        sort:
          method: friendly_name
          reverse: false
`

As you can see, i used an "input_select" with the different media_players to select the speaker to play the audio from. The final result looks like this: 
<img width="289" alt="example_ma_dashboard" src="https://github.com/user-attachments/assets/b004a980-8000-4087-88e4-d51658344bae" />

This overall is just a really simplification of some service calls behind a card. But this works flawless for me in combination with normal media cards to take over the control part.

Bests
